plugins/dates/src/model/regex.js

```diff
@@ -10,7 +10,7 @@ export default [
   // @4pm
   [/^@[1-9]+(am|pm)$/, 'Time', '@5pm'],
   // 03/02
-  [/^[0-9]{2}\/[0-9]{2}$/, 'Date', '03/02'],
+  [/^(?:0[1-9]|1[0-2])\/(?:0[1-9]|[12]\d|3[01])$/, 'Date', '03/02'],
```

Comment:
given this info (https://github.com/spencermountain/compromise/pull/1129#issuecomment-2248977259), maybe this:
```suggestion
  [/^(?:0[1-9]|[12]\d|3[01])\/(?:0[1-9]|[12]\d|3[01])$/, 'Date', '03/02'],
```

Reply:
https://github.com/spencermountain/compromise/pull/1133/files